src/util.go

```diff
@@ -80,11 +80,23 @@ func init() {
 	rand.Seed(time.Now().Unix())
 }
 
+func lastField(r string) string {
+    ss := strings.Fields(r) // seperated string
+    return ss[len(ss) - 1] // get last word
```

Comment:
You should always make sure len(ss) != 0 when writing a generic function like this. In this case we know the caller will always present an r with at least one word, but in general you should make no assumptions.

Reply:
Sorry, I am quite new to golang, how would I make sure that it is more than 0? Is there like some kind of exit function that stops the process?